jenkins/artifact-signer/Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
+            alwaysPull true
+        }
+    }
+    parameters{
+        string(
+                name: 'ARTIFACT_PATH',
+                trim: true,
+                description: 'Path to the artifact or the artifact directory on s3'
+        )
+        choice(
+                choices: ['linux'],
+                name: 'DISTRIBUTION_PLATFORM',
+                description: 'What platform is this distribution build for?'
+        )
+        choice(
+                choices: ['.sig'],
+                name: 'SIGNATURE_TYPE',
+                description: 'What is signature file type?'
+        )
+    }
+    stages {
+        stage('sign') {
+            steps {
+                script {
+                    if (ARTIFACT_PATH == '') {
+                        currentBuild.result = 'ABORTED'
+                        error('ARTIFACT_PATH is not set')
+                    }
+                    ARTIFACT_PATH = ARTIFACT_PATH.replaceAll('/$', "")
```

Comment:
What does this do?

Reply:
Removes the ending forward slash, if added, since we too add a forward slash when pulling and completing operations when creating path etc. Having 2 slashes creates unnecessary problems, folder named `/`, problems in `s3Download` etc.